packages/zoltan/src/zz/zz_const.h

```diff
@@ -251,6 +251,14 @@ struct Zoltan_Struct {
                                       usually 0 (no weights) or 1            */
   int Timer;                      /*  Timer type that is currently active */
   struct Zoltan_Timer *ZTime;     /*  Timer structure for persistent timing. */
+
+  struct Zoltan_Struct *Highest_Ancestor_ZZ; /* ZZ Struct used for hier 
```

Comment:
Please describe all added fields.  What do they do, what is their contents, etc.  What is this pointer pointing to?  What is the "Highest Ancestor"?